typing/types.ml

```diff
@@ -44,7 +44,11 @@ and type_desc =
   | Tvariant of row_desc
   | Tunivar of string option
   | Tpoly of type_expr * type_expr list
-  | Tpackage of Path.t * (string list * type_expr) list
+  | Tpackage of package
+
+and package =
+    { pack_path : Path.t;
+      pack_fields : (string list * type_expr) list }
```

Comment:
The name `pack_fields` feels too generic. Maybe  `pack_constraints` (or `pack_cstrs`) ?

Reply:
I've just done the proposed renaming. For coherence I also renamed the field in `package_type` (defined in `typedtree.ml`) from `pack_fields` to `pack_cstrs`.

We currently have a collision of record fields between `package` in `types` and `package_type` in `typedtree`. Should we change the names in `package_type` to something like `tpt_*` or is this collision acceptable ?

Reply:
Collisions in different modules are acceptable because the field names can be disambiguated with `Types.pack_path` or ` Typedtree.pack_path`.